.github/workflows/build.yml

```
@@ -0,0 +1,20 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  lint-server:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

```suggestion
      - uses: actions/checkout@v5
```
action checkout seems to have new version:
https://github.com/actions/checkout